pkg/cmd/grafana-server/main.go

```diff
@@ -56,16 +58,56 @@ func main() {
 		os.Exit(0)
 	}
 
-	if *profile {
+	ultimateProfileEnabled := *profile
+	ultimateProfilePort := *profilePort
+	profileEnv, exists := os.LookupEnv("GF_PROCESS_PROFILE")
+	if exists {
+		enabled, parseErr := strconv.ParseBool(profileEnv)
+		if parseErr != nil {
+			panic(parseErr)
+		} else {
```

Comment:
I think it's more idiomatic in Go to drop the `else`, since you won't get here if there's an error.